packages/twenty-front/src/modules/activities/calendar/components/CalendarEventDetails.tsx

```diff
@@ -111,10 +113,14 @@ export const CalendarEventDetails = ({
       >
         <RecordFieldComponentInstanceContext.Provider
           value={{
-            instanceId: getRecordFieldInputId(calendarEvent.id, fieldName),
+            instanceId: getRecordFieldInputId({
+              recordId: calendarEvent.id,
+              fieldName,
+              prefix: INPUT_ID_PREFIX,
+            }),
           }}
         >
-          <RecordInlineCell readonly />
+          <RecordInlineCell inputIdPrefix={INPUT_ID_PREFIX} />
```

Comment:
neat: we could assume that RecordInlineCell is always in a RecordFieldComponentInstanceContext and get the instanceId from there